README.md

```diff
@@ -1,3 +1,16 @@
 # Semantic Version Comparison in a Jenkins pipeline
 
 A Jenkins global pipeline library method [semver_compare](vars/semver_compare.groovy) and an example [Jenkinsfile](Jenkinsfile) for how to use it.
+
+## Running tests
```

Comment:
The [rendered version of this section](https://github.com/etoews/jenkins-semver-compare/blob/4f5c601dedb3557f621bf57eef5fde5091045e82/README.md) looks a bit off.

Please use the following

```markdown
Gradle is used to run tests for the SemanticVersion class used in this library.
The repository comes with [Gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html), a helper script to automatically download and manage the gradle distribution. Invoke the wrapper script according to the OS platform:

UNIX/Linux

    ./gradlew test

Windows (untested)

    gradlew.bat
```

Reply:
Ooops, sorry for that oversight. I initially blamed the preview in IDEA, but it does not look right there either so this is purely on me.

I'd like to keep the "one line per sentence" convention, as it does not affect the output and makes future diffs easier to read - the rest of the suggested changes are applied.